client/src/sections/learning-paths/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useGetLearningPathsQuery } from '@/services/learningPathAPI';
+import LearningPathCard from '@/src/shared/components/Card/LearningPathCard';
+import Pagination from '@/src/shared/components/Pagination';
+import { alertError, type LearningPath } from '@/src/shared/utils';
+
+interface LearningPathListProps {
+  isActive: boolean;
+  page: number;
+  handleChangePageEvent: (page: number) => void;
+}
+
+const LearningPathList = ({
+  isActive,
+  page,
+  handleChangePageEvent,
+}: LearningPathListProps): JSX.Element => {
+  const { data, error, isLoading } = useGetLearningPathsQuery({
+    page,
+    isActive,
+  });
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return alertError('An error occurred');
+  }
+
+  const { results, totalPages, current_page_number: currentPage } = data;
+  return (
+    <div className="flex flex-col justify-center w-full">
+      {results.length ? (
+        <>
```

Comment:
```suggestion
        <Fragment>
```

please also apply this to 
https://github.com/framgia/sph-lms/pull/124/files#diff-de05f53a2c0f62dc2668b4ed96ad5668cb2b5e618270c8b82a4ea8d33dc1323a